x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/diff_components/index_patterns.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBadge, EuiFlexGroup, EuiFlexItem, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/css';
+import React, { useMemo } from 'react';
+import { RuleDiffPanelWrapper } from './panel_wrapper';
+
+export interface IndexPatternDiffComponentProps {
+  oldIndexPatterns: string[];
+  newIndexPatterns: string[];
+}
+
+export const IndexPatternDiffComponent = ({
```

Comment:
This is unused right now, right? Let's just remove it from the PR for now, we can always retrieve it later as long as we don't delete the branch from Github.